DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml

```diff
@@ -0,0 +1,31 @@
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@model FaqsPageViewModel
+
+<h2 class="side-nav__heading">System</h2>
```

Comment:
This file should be CRLF line endings.

Reply:
I think these are all sorted now, went through everything on this branch and changed any LF to CRLF. Sorry for giving you the run around on this!

Reply:
Nice, looks like that is all sorted now. No worries, glad this could get sorted.